drivers/input/Kconfig.gt911

```diff
@@ -26,4 +26,10 @@ config INPUT_GT911_INTERRUPT
 	help
 	  Enable interrupt support (requires GPIO).
 
+config INPUT_GT911_MAX_TOUCH_POINTS
+	int "Touch Number"
+	default 5
```

Comment:
I'd consider making this dependent on `CONFIG_INPUT_GT911_MULTITOUCH_ENABLED`, and just setting the touch points register to 0b1 if multitouch support isn't enabled.

Reply:
Following the same logic, it's a good idea. I hope I changed it the way you asked.